scripts/prox_crosswalk.py

```diff
@@ -40,9 +38,36 @@
 # HACK: used to avoid duplicating Yelp place requests.
 _YELP_ID_TO_PLACE_CACHE = {}
 
-
-def _get_crosswalk_db(): return _firebase.database().child(_CROSSWALK_PATH)
-
+def getSourceIDs(keyID, sourcesList, identifiers):
```

Comment:
It's not clear that this will call a function that has the side effect of writing to the DB – can you change the name to make that clearer? e.g. getSourceIDsAndCacheSources

Personally, I try to limit my functions to either getting or setting for clarity.